ios/sdk/src/JitsiMeetView.m

```diff
@@ -136,7 +139,9 @@ +             (BOOL)application:(UIApplication *)application
   didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Store launch options, will be used when we create the bridge.
     _launchOptions = [launchOptions copy];
-
+    
+    [Dropbox setAppKey];
```

Comment:
AFAICT from reading the Dropbox SDK code it doesn't seem absolutely necessary to call this so early. Can you double check? I think it would greatly simplify things if we did this later, from JS.

Reply:
Sounds dangerous to me. In the dropbox docs they are using it here...

Reply:
@hristoterezov, where is the documentation for that please? I must be dumber than usual today, I'm sorry, and I cannot find it. `JitsiMeetView` seems like an inappropriate place to do something that's app-related, not View specific.